composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^5.3.6 || >=7.0 <7.2",
+        "doctrine/annotations": "^1.2",
```

Comment:
no, version you requested is 1.2, which will work on all versions that we do support.
while building phar, we set up platform config for composer, so it will install newest version that will still work on lowest supported platform.